src/cli/cli.cpp

```diff
@@ -5411,6 +5411,168 @@ template <> otError Interpreter::Process<Cmd("nexthop")>(Arg aArgs[])
 exit:
     return error;
 }
+
+#if OPENTHREAD_CONFIG_MESH_DIAG_ENABLE
+
+template <> otError Interpreter::Process<Cmd("meshdiag")>(Arg aArgs[])
+{
+    otError error = OT_ERROR_NONE;
+
+    if (aArgs[0] == "topology")
```

Comment:
We should add Doxygen tags for the CLI reference by default whenever a new command is added.  My team can do this later, though, if this needs to get in sooner.

Reply:
Added copy of docs from `README.md` here. It contains bullet point list with sub-lists - not sure if sub-lists will work and look ok. If not we can fix later.

Reply:
Yes they will work.  Added a few comments to complete the documentation for the command.